FND/DatesAndTimes/BusinessDates.rdf

```diff
@@ -135,26 +136,18 @@ They are modularized this way to minimize the ontological committments that are
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>business recurrence interval</rdfs:label>
-		<skos:definition>a recurrence interval that is specified using a business recurrence interval convention</skos:definition>
+		<skos:definition>recurrence interval that is specified using a business recurrence interval convention</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-dt-bd;BusinessRecurrenceIntervalConvention">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-bd;Convention"/>
 		<rdfs:label>business recurrence interval convention</rdfs:label>
-		<skos:definition>a convention for modeling recurring days, such as the end of the month, a particular day of the month, a day of the week, or more specifically, a t-bill auction date</skos:definition>
+		<skos:definition>convention that specifies how recurring days should be handled, such as the end of the month, a particular day of the month, a day of the week, or more specifically, a t-bill auction date</skos:definition>
 	</owl:Class>
 	
-	<owl:NamedIndividual rdf:about="&fibo-fnd-dt-bd;BusinessRecurrenceIntervalEndOfMonth">
```

Comment:
Should this not be deprecated and made equivalentTo EndOfMonth?

Reply:
We had agreed that for FIBO 2.0 we were not deprecating anything until after the FTF work is done.